### PR TITLE
Add management command for syncing Firebase UIDs & fetching tokens

### PR DESCRIPTION
**Pull request by brittanylau**
### Notion ticket link

[Spring cleaning 🌼 ](https://www.notion.so/uwblueprintexecs/9cb454262ed84c6299c0682c23f92ec5?v=1076bd30711a4a01838becee05f8b5a3&p=4b8bb43e63664dceaf6a582dfe7d0806)

### Implementation description

- created a management command that accepts an email & password, and:
  1. populates the user's Firebase UID
  2. prints a valid auth token to use
- note: the command assumes that your Django and Firebase users have the same email (will be mandatory for auth later)

### Steps to test

1. ensure a local user exists with the same email as your Firebase account
1. `python manage.py sync_firebase_user [email] [password]` 

### What should reviewers focus on?

- does it work?
- would it be helpful for this script to do anything else?

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: accounts/management/commands/sync_firebase_user.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    help = "Syncs a Firebase user with a Django user"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "email",
+            type=str,
+            nargs=1,
+            help="Firebase & Django account email",
+        )
+        parser.add_argument(
```
Comment (danielchen115) [inline accounts/management/commands/sync_firebase_user.py]: nit: can we hide the password when it's being entered?

File: accounts/management/commands/sync_firebase_user.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                "email": email,
+                "password": password,
+                "returnSecureToken": True,
+            },
+        ).json()
+
+        firebase_uid = json["localId"]
+        token = json["idToken"]
+
+        user = User.objects.get(email=email)
```
Comment (danielchen115) [inline accounts/management/commands/sync_firebase_user.py:34]: itd be nice if we added some exception handling, but if it's just being used by us then it'll be fine

**Review by danielchen115 — APPROVED** [review]
lgtm :) Just some nits

File: accounts/management/commands/sync_firebase_user.py (hunk 1 above)
Comment (brittanylau) [reply accounts/management/commands/sync_firebase_user.py]: good idea, done :)

File: accounts/management/commands/sync_firebase_user.py (hunk 2 above)
Comment (brittanylau) [reply accounts/management/commands/sync_firebase_user.py:34]: yeah it'll just be us using it - I think it'd help us diagnose seeing the real error if something does go wrong

**Pull request merged**